ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -160,11 +174,76 @@ export default class ConfirmPageContainerContent extends Component {
         {this.renderContent()}
         {!supportsEIP1559V2 &&
           !hasSimulationError &&
-          (errorKey || errorMessage) && (
+          (errorKey || errorMessage) &&
+          currentTransaction.type !== TRANSACTION_TYPES.SIMPLE_SEND && (
             <div className="confirm-page-container-content__error-container">
               <ErrorMessage errorMessage={errorMessage} errorKey={errorKey} />
             </div>
           )}
+        {supportsEIP1559V2 &&
+          !hasSimulationError &&
+          (errorKey || errorMessage) &&
+          errorKey === INSUFFICIENT_FUNDS_ERROR_KEY &&
+          currentTransaction.type === TRANSACTION_TYPES.SIMPLE_SEND && (
```

Comment:
[nit] Feels like this conditional chain should maybe be extracted into a `const`? `showInsuffienctFundsError`?

Reply:
done.